### DISCUSSION TITLE
Adding product reviews

### DISCUSSION THREAD

**Discussion by adedayo14** [swellstores/community#570] [Help] [answered]
Could anyone share how they’ve successfully added product reviews to their Swell store, allowing customers to add and view others' reviews?

Many thanks

**Comment by sags95** [comment, marked as answer]
We do have a sample Swell App that would allow this: https://github.com/swellstores/honest-reviews-app. If you're interested in testing Swell Apps then simply reach out to apps-alpha@swell.is with your store-id and we can enable access.
